Detections/DnsEvents/DNS_TorProxies.yaml

```diff
@@ -1,7 +1,7 @@
 ﻿id: a83ef0f4-dace-4767-bce3-ebd32599d2a0
-name: DNS tor proxies
+name: DNS event for ToR proxies
```

Comment:
maybe use "related" instead of "for" in the name.  So, Name would be "DNS events related to ToR proxies".  This would go in line with what we have for the Miners detection also and I think is more accurate.